### PR TITLE
Drop chrome-common-chrome_content_client_constants.cc.patch

### PR DESCRIPTION
**Pull request by mariospr**
This patch can be removed in favour of chromium_src overrides to
redefine the values for the constants we're interested in.

Resolves https://github.com/brave/brave-browser/issues/14338

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

See Test plan for https://github.com/brave/brave-core/pull/2342

### CODE REVIEW

**Comment by mariospr** [comment]
All the relevant bots have passed already(*) so moving this to the review process:

* @mkarolin PTAL on behalf of @brave/patch-reviewers 
* @simonhong PTAL as the original author

Thanks!

(*) The failure on the post-init bot is unrelated (being currently worked on separately)

File: chromium_src/chrome/common/chrome_content_client.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_COMMON_CHROME_CONTENT_CLIENT_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_COMMON_CHROME_CONTENT_CLIENT_H_
+
+#include "chrome/common/chrome_content_client.h"
```
Comment (mkarolin) [inline chromium_src/chrome/common/chrome_content_client.h:9]: This #include doesn't seem right.

Comment (mariospr) [reply chromium_src/chrome/common/chrome_content_client.h:9]: Oops! Removed now, thanks

**Review by simonhong — APPROVED** [review]

**Comment by mariospr** [comment]
@simonhong Thanks for the review
@mkarolin Still need an owner's review, PTAL when you have a moment, thanks!

**Comment by mariospr** [comment]
> @mkarolin Still need an owner's review, PTAL when you have a moment, thanks!

@mkarolin Ping?

**Review by mkarolin — APPROVED** [review]
++

**Pull request merged**